specs/signature-specification.md

```diff
@@ -207,6 +209,14 @@ Besides the [image manifest property requirements][image-manifest-property-descr
   Keys using the `io.cncf.notary` namespace are reserved for use in the Notary Project signature specification and MUST NOT be used by other specifications.
   - **`io.cncf.notary.x509chain.thumbprint#S256`**: A REQUIRED annotation whose value contains the list of SHA-256 fingerprints of signing certificate and certificate chain (including root) used for signature generation. The list of fingerprints is present as a JSON array string, corresponding to ordered certificates in [*Certificate Chain* unsigned attribute](#unsigned-attributes) in the signature envelope. The annotation name contains the hash algorithm as a suffix (`#S256`) and can be extended to support other hashing algorithms in future.
 
+For OCI v1.0 compliant registries, the `subject` and `artifactType` fileds in the signature manifest are ignored by the registry. Notation v2.x SHOULD fall back to use OCI referrers tag schema to push and store the signature in the registry.
```

Comment:
`OCI v1.0` is not accurate, please specify whether it is image-spec or distribution-spec or both.